tools/generate_point_cloud.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+###############################################################################
+# Copyright Kitware Inc. and Contributors
+# Distributed under the Apache License, 2.0 (apache.org/licenses/LICENSE-2.0)
+# See accompanying Copyright.txt and LICENSE files for details
+###############################################################################
+
+import os
+import sys
+import argparse
+import subprocess
+from ply2txt import convert
+#from VisSatSatelliteStereo.stereo_pipeline import StereoPipeline
+
+def main(args):
+    parser = argparse.ArgumentParser(
+        description='Generate LAS file from satellite images.')
+    parser.add_argument('--config_file',
+                        help='JSON file to configure VisSat', required=True)
+    parser.add_argument('--work_dir',
+                        help='Work directory for VisSat', required=True)
+    parser.add_argument('--point_cloud',
+                        help='Output LAS file', required=True)
+    parser.add_argument('--utm', 
+                        help='UTM zone that AOI is located in', required=True)
+    args = parser.parse_args(args)
+
+    #pipeline = StereoPipeline(args.config_file)
+    #pipeline.run()
+    subprocess.run(["python3", "/VisSatSatelliteStereo/stereo_pipeline.py", 
+                    "--config_file", args.config_file], check=True)
+    
+    convert([os.path.join(args.work_dir, 
+            'mvs_results/aggregate_3d/aggregate_3d.ply'), 
+            os.path.join(args.work_dir, 
+            'mvs_results/aggregate_3d/aggregate_3d.txt')])
+
+    subprocess.run(["/LAStools/bin/txt2las", 
+    				"-i", os.path.join(args.work_dir,
```

Comment:
looks like you may have a mix of tabs and spaces.  Use just spaces and line these lines up.